examples/full_fledged_schema_examples_new/stratification/sir_linear_stratification.ipynb

```diff
@@ -663,7 +661,7 @@
     "    \n",
     "    :flows\n",
     "    CLOUD => f_NewBorn(v_NewBorn) => NormalWeight\n",
-    "    NormalWeight => f_DeathNormalWeight(v_DeathNormalWeight) => ClOUD\n",
+    "    NormalWeight => f_DeathNormalWeight(v_DeathNormalWeight) => CLOUD\n",
     "    NormalWeight => f_BecomingOverWeight(v_BecomingOverWeight) => OverWeight\n",
```

Comment:
I swear, I thought we've fixed this same thing 3 times over...

Reply:
Now that we have CI on these, hopefully it never slips back in! 😃